components/menu/test/menu-items/menu-item-mixin.html

```diff
@@ -45,6 +45,36 @@
 						});
 						menuItem.click();
 					});
+
+					it('fires select event when enter key pressed on item', (done) => {
```

Comment:
Similar to other places, I'd probably switch to `dispatch` term, since it matches the browser term, and `fire` was sort of a Polymer thing.